lib/verify-fqdn.sh

```diff
@@ -90,7 +90,7 @@ dns_verify() {
   output "Resolving DNS for $fqdn"
   ip=$(curl -4 -s $CHECKIP_URL)
   dns_record=$(dig +short @$DNS_SERVER "$fqdn")
-  [ "${ip}" != "${dns_record}" ] && fail
+  [[ $dns_record != *"$ip"* ]] && fail
```

Comment:
Why have they switched places?

Reply:
No real reason, I just quickly typed that. 
I hardly ever type shell, but I think the "'s can also be removed around $ip.